tests/installation/disable_grub_timeout.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Disable grub timeout from the Installer
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run {
+    my ($self) = shift;
+
+    # Verify Installation Settings overview is displayed as starting point
+    assert_screen "installation-settings-overview-loaded";
+
+    if (check_var('VIDEOMODE', 'text')) {
+        # Select section booting on Installation Settings overview on text mode
+        send_key 'alt-c';
+        assert_screen 'inst-overview-options';
+        wait_screen_change { send_key 'alt-b'; };
+    }
+    else {
+        # Select section booting on Installation Settings overview (video mode)
+        send_key_until_needlematch 'booting-section-selected', 'tab', 10;
+        assert_screen 'booting-section-selected';
+        send_key 'ret';
+    }
+
+    # Select bootloader options tab
+    wait_screen_change { send_key 'alt-r'; };
+    assert_screen 'installation-bootloader-options';
+
+    # Select Timeout dropdown box and increase its value
```

Comment:
well, "increase" does not really apply here anymore, does it? ;-)

Reply:
good catch!